gallery_dl/extractor/lightroom.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018-2022 Mike Fährmann
```

Comment:
I'm not the author or copyright holder.
Put your own name and *current year* there, or just delete this line.